knowledge/miscellaneous_unknown/qna.yaml

```diff
@@ -0,0 +1,64 @@
+created_by: gtyellow
+version: 3
+domain: Sports
+document_outline: >-
```

Comment:
The `document_outline` should be a "one liner" descriptive overview of the submission. This should be concise yet descriptive enough for an LLM to build context.